.github/workflows/undeploy-2-start.yml

```diff
@@ -16,6 +16,27 @@ jobs:
     runs-on: ubuntu-latest
     environment: ${{ inputs.deployment-environment }}
     steps:
+      - uses: actions/checkout@v4
+
+      - name: Get Versions From config/versions.json
+        id: versions
+        run: |
+          echo "spack=$(jq --compact-output --raw-output '.spack' ./config/versions.json)" >> $GITHUB_OUTPUT
+          echo "config=$(jq --compact-output --raw-output '."spack-config"' ./config/versions.json)" >> $GITHUB_OUTPUT
+
+      - name: Get ${{ inputs.deployment-environment }} Remote Paths
+        id: path
+        run: |
+          if [ -z "${{ vars.SPACK_INSTALLS_ROOT_LOCATION }}" ]; then
+            echo '::error::`vars.SPACK_INSTALLS_ROOT_LOCATION` does not exist in `${{ github.repository }}`s `${{ inputs.deployment-environment }}` environment'
+            exit 1
+          fi
+
+          root=${{ vars.SPACK_INSTALLS_ROOT_LOCATION }}/${{ steps.versions.outputs.spack }}
+          echo "root=$root" >> $GITHUB_OUTPUT
+          echo "spack=$root/spack" >> $GITHUB_OUTPUT
+          echo "spack-config=$root/spack-config" >> $GITHUB_OUTPUT
+
```

Comment:
Sorry to be *that* guy, but isn't this a copy of the code from `deploy-2-start.yml`? Could it be defined once and reused?

Reply:
I might make it an action - that would be the most reusable thing

Reply:
Done in https://github.com/ACCESS-NRI/build-cd/pull/96/commits/64193d6bff2008c5170e95d0e565de7ea09044d6